heft-plugins/heft-dev-cert-plugin/src/DevCertPlugin.ts

```diff
@@ -10,7 +10,9 @@ import type {
   IScopedLogger
 } from '@rushstack/heft';
 import { CertificateManager, ICertificate } from '@rushstack/debug-certificate-manager';
-import { Configuration as WebpackDevServerConfig } from 'webpack-dev-server';
+
+// IMPORTANT: To simplify versioning, 'webpack-dev-server' is a devDependency, not a regular dependency
+import type { Configuration as WebpackDevServerConfig } from 'webpack-dev-server';
```

Comment:
Why is it important for the reader to know, in this context, that `'webpack-dev-server'` is a `devDependency`? I see "IMPORTANT:", and a fact -- but no explanation of why that fact is important, or what I-the-reader need to do about it.

My apologies if this question merely reflects my own ignorance, as is entirely possible.

Reply:
If you try to import the value and reference it, it will compile and work locally, but fail at runtime when published. Only the type information is safe to use, hence the choice of type-only import.

Reply:
@MickeyPhoenix Right, normally when you import from a package, that package must be a regular dependency (since a dev dependency will be missing for library consumers, as David explained). So the `IMPORTANT` note is explaining why we are doing `import type`. 

It's a common situation for published libraries, but I can add a bit more context to the comment.